ui/node_menu.py

```diff
@@ -531,8 +532,13 @@ def draw(self, context):
         insertNode(layout, "an_IntersectLinePlaneNode", "Intersect Line Plane")
         insertNode(layout, "an_IntersectLineSphereNode", "Intersect Line Sphere")
         insertNode(layout, "an_IntersectPlanePlaneNode", "Intersect Plane Plane")
+        insertNode(layout, "an_IntersectSpherePlaneNode", "Intersect Plane Sphere")
+        insertNode(layout, "an_IntersectSphereSphereNode", "Intersect Sphere Sphere")
         layout.separator()
         insertNode(layout, "an_IntersectPolylinePlaneNode", "Intersect Polyline Plane")
+        layout.separator()
+        insertNode(layout, "an_BarycentricTransformNode", "Barycentric Transform")
+        insertNode(layout, "an_BarycentricTransformNode", "Barycentric Transform List", {"operationType" : repr("LIST")})
```

Comment:
I was imitating vector transform / transform list
for me that comes handy sometimes, and here as default is not list
however, if u want it out, out it is